src/main/java/uk/ac/ebi/eva/pipeline/parameters/validation/job/GenotypedVcfJobParametersValidator.java

```diff
@@ -52,6 +53,8 @@ private CompositeJobParametersValidator compositeJobParametersValidator(JobParam
 
         Boolean skipAnnotation = Boolean.valueOf(jobParameters.getString(JobParametersNames.ANNOTATION_SKIP));
         if (!skipAnnotation) {
+            jobParametersValidators.add(new InputStudyIdValidator());
```

Comment:
yes, I suggested to put the studyIdValidator as a required parameter to the stepValidator constructor (converting a logical dependency into a syntactical dependency) but with @jorizci we decided this was simpler. I'll add a comment. 

I still prefer my suggestion, it avoids logical programming errors without the need of comments for clarity